common/src/config.rs

```diff
@@ -32,7 +32,7 @@ impl Default for Database {
     fn default() -> Self {
         const DEFAULT_PORT: u16 = 5432;
         Database {
-            username: env::var("DB_USER").unwrap_or("trustify".into()),
+            username: env::var("DB_USER").unwrap_or("postgres".into()),
```

Comment:
We changing credentials?

Reply:
IDU. We're changing the default username.